NitroxLauncher/Pages/ServerPage.xaml.cs

```diff
@@ -1,37 +1,848 @@
+                InitializeWorldListing();
+            }
+            else
+            {
+                SaveConfigSettings();
+                InitializeWorldListing();
+                IsNewWorld = false;
+                IsInSettings = false;
+
+                ImportSaveBtnBorder.Opacity = 0;
+                ImportSaveBtn.IsEnabled = false;
+            }
+
+            Storyboard GoBackAnimationStoryboard = (Storyboard)FindResource("GoBackAnimation");
+            GoBackAnimationStoryboard.Begin();
+        }
+
+        private void ImportSaveBtn_Click(object sender, RoutedEventArgs e)
+        {
+            ImportedWorldName = string.Empty;
+            SelectedWorldImportDirectory = string.Empty;
+            SelectedServerCfgImportDirectory = string.Empty;
+            TBImportedWorldName.Text = string.Empty;
+            TBSelectedSaveImportDir.Text = "Select the save file to import";
+            TBSelectedServerCfgImportDir.Text = "Select the server.cfg file to import";
+
+            ImportWorldBox.Opacity = 1;
+            ImportWorldBox.IsHitTestVisible = true;
+            SelectImportedServerCfgBtn.IsEnabled = false;
+            SelectImportedServerCfgBtn.Opacity = .6;
+            ImportWorldBtn.IsEnabled = false;
+            ImportWorldBtn.Opacity = .6;
+        }
+
+        // World management
+        private void SelectedWorldSettings_Click(object sender, RoutedEventArgs e)
+        {
+            if (LauncherLogic.Server.IsServerRunning && WorldCurrentlyUsed == WorldManager.GetSaves().ElementAtOrDefault(WorldListingContainer.SelectedIndex)?.WorldSaveDir)
+            {
+                LauncherNotifier.Error("This world is currently being used. Stop the server to edit the settings of this world");
+                return;
+            }
+            else if (!Directory.Exists(WorldManager.GetSaves().ElementAtOrDefault(WorldListingContainer.SelectedIndex)?.WorldSaveDir))
+            {
+                LauncherNotifier.Error($"This save does not exist or is not valid.");
+                InitializeWorldListing();
+                return;
+            }
+            else if (!WorldManager.GetSaves().ElementAtOrDefault(WorldListingContainer.SelectedIndex).IsValidSave)
+            {
+                LauncherNotifier.Error($"This save is an invalid version.");
+                return;
+            }
+
+            TBWorldSeed.IsReadOnly = true;
+
+            SelectedWorldDirectory = WorldManager.GetSaves().ElementAtOrDefault(WorldListingContainer.SelectedIndex)?.WorldSaveDir ?? "";
+
+            UpdateVisualWorldSettings();
+
+            Storyboard WorldSelectedAnimationStoryboard = (Storyboard)FindResource("WorldSelectedAnimation");
+            WorldSelectedAnimationStoryboard.Begin();
+        }
+
+        // Restore Backup Button(WIP)
+        //private void RestoreBackup_Click(object sender, RoutedEventArgs e)
+        //{
+
+        //}
```

Comment:
Code should not be commented. If it isn't implemented yet, throw a `throw new NotImplementedException();` with a TODO comment so it's easy to search the code base for it.